website/source/docs/job-specification/service.html.md

```diff
@@ -120,6 +122,11 @@ the script will run inside the Docker container. If your task is running in a
 chroot, it will run in the chroot. Please keep this in mind when authoring check
 scripts.
 
+- `address_mode` `(string: "host")` - Same as `address_mode` on `service`.
```

Comment:
I would add a full example for this and link it to this

Reply:
Added a full section at the bottom of that page with a link